### DISCUSSION TITLE
Problem with building the project

### DISCUSSION THREAD

**Discussion by minkb83** [DevSkillsHQ/discusssions#52] [Help] [answered]
Hi,
I have issues both cloning the repo into my local machine and building it on the web.
Locally i can't clone. On the web, it doesn't build the projetc complaining about missing header files.

**Comment by fenske** [comment]
Hi @minkb83! I'm Anton from Alva. I couldn't find your repository. Could you either share the link to your repo or, if you used a different GitHub username, tell me what it was?

**Comment by minkb83** [comment]
Hi Anton,

Yes I changed my username, the old one was minookargar.

And this is the link
https://github.com/ffc1e12/uart_level3_c_0710f44-58i4cj

Best
Minoo

**Comment by ludvig** [comment, marked as answer]
Hi @minkb83 we have committed a change in your repository and it should work now to re-create your codespaces instance from latest main branch.

**Comment by minkb83** [comment]
Hi Ludvig,

The update fixed the issue,
thank you

Best
Minoo
